docs/content/what-is-continue-as-new.md

```diff
@@ -0,0 +1,30 @@
+---
+id: what-is-continue-as-new
+title: What is Continue As New
+sidebar_label: Continue As New
+description: Continue-As-New is the mechanism by which all relevant state is passed to a new Workflow Execution with a fresh Event History.
+tags:
+  - explanation
+  - continue-as-new
+---
+
+Continue-As-New is the mechanism by which all relevant state is passed to a new Workflow Execution with a fresh Event History.
+
+The reason why this exists is to prevent a Workflow Execution Event History from becoming too large.
+As a precautionary measure, the Temporal Platform limits Event Histories to 50,000 and will warn you every 10,000 Events.
+
```

Comment:
Suggestion:
"Continue-As-New is a mechanism by which the latest relevant state is passed to a new Workflow Execution, with a fresh Event History. 
As a precautionary measure, the Temporal Platform limits Event History to 50,000 Events, and will warn you every 10,000 Events. To prevent a Workflow Execution Event History from exceeding this limit and failing, use Continue-As-New to start a new Workflow Execution instance with a fresh Event History."